apps/test/unit/templates/ImageWithStatusTest.js

```diff
@@ -88,6 +90,7 @@ describe('ImageWithStatus', () => {
       // ImageWithStatus, so wait until the next clock tick before checking
       // expectations.
       setTimeout(function() {
+        root.update();
```

Comment:
Root cause: [For mount, updates are sometimes required when they weren't before](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#for-mount-updates-are-sometimes-required-when-they-werent-before). In particular, the image load event and timeout here don't trigger an update by enzyme, so we have to tell it that one should occur ourselves.